docs/engineering/Certificates.md

```diff
@@ -0,0 +1,79 @@
+---
+id: certificates
+title: Certificates
+---
+
+# Certificates
+
+SSL encryption for all services are provided by LetsEncrypt certificates that are generated and shared by an AWS instance named "certbot"
+
+## certbot
+
+[cerbot instance on AWS](https://us-west-1.console.aws.amazon.com/ec2/v2/home?region=us-west-1#InstanceDetails:instanceId=i-0e3e24e76461de3f1)
+
+LetsEncrypt/Certbot is configured on the "certbot" Ubuntu instance and uses the certbot Cloudflare plugin to automatically verify ownership with a Cloudflare API key.
+
+### Cloudflare API Key
+
+To change or reference the key, it can be found at 
+
+```bash
+/etc/cloudflare/credentials.ini
+```
+---
+
+### Altering Certificate Registration
+
+The existing certificate is registered with the command:
+
+```bash
+sudo certbot certonly \
+  --dns-cloudflare \
+  --dns-cloudflare-credentials /etc/cloudflare/credential.ini \
+  -d *.webaverse.com
+  ```
+
+To add to or change the domains included in the certificate, add to or change the ```-d ...``` row(s) and run again.
+
+---
+
+### Certificate Renewal
+
+LetsEncrypt certificates expire after only 3 months, but the Certbot tool can automatically renew certificates when expiration nears.
+
+---
+
+### Scripts
+
+We have a script that is scheduled to run automatically every Monday that not only runs the certbot renewal command, but also copies the latest certificate files to an S3 bucket and registers the certificate with AWS Certificate Manager.
+
+```bash
+/scripts/certbot_renewal.sh
```

Comment:
Is the below supposed to be this script? Or is `/scripts/certbot_renewal.sh` found somewhere?